Borba02/nss_Timed_Trials
# Stop Tracking a UserTrial

#### Changes Made
1. Added button to userTrial List on each userTrial card that allows user to show a userTrial as deactivated
3. Added Controller method that gets the UserTrial from Id and sets the SubscriptionActive to false 
4. Added Repository method and interface for updating a UserTrial
5. Removed Delete Trial button and updated SQL for the trial list to make expired trials fall off the trial list
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bb-EndingAUserTrial
    ```
7. Open a new Terminal tab (⌘T) and navigate to the server directory.
8. Test app functionality.
    >GIVEN a user would like to record that they have cancelled their trial subscription
AND they are on the home page view
WHEN they click on the "Subscription Ended" button on a trial card
THEN the trial will be marked as inactive
9. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

closes issue #7
